docs/content/how-to/openstack/prerequisites.md

```diff
@@ -0,0 +1,105 @@
+# Prerequisites for OpenStack
+
+* Admin access to an OpenShift cluster (version 4.17+) specified by the `KUBECONFIG` environment variable.
+  This cluster is referred to as the Management OCP cluster.
+* The Management OCP cluster can be running on OpenStack (ShiftOnStack), but it could also be running on Baremetal or
+  a public cloud such as AWS (what our CI does).
```

Comment:
```suggestion
  a public cloud such as AWS.
```
Isn't our ci already doing both, aws and osp?

Reply:
yeah.